src/main/resources/templates/dashboard.html

```diff
@@ -45,7 +74,20 @@ <h2 class="position-absolute top-50 start-50 translate-middle"><strong>No Produc
           <img th:if="${product.image == null}" th:src="@{/images/placeholder.jpg}" src="../static/images/placeholder.jpg" class="card-img-bottom mb-2" alt="Placeholder">
           <img th:unless="${product.image == null}" th:src="@{/images/placeholder.jpg}" src="../static/images/placeholder.jpg" class="card-img-bottom mb-2" alt="Placeholder">
           <h5 class="card-title" th:text="|${product.name}|">Product Name</h5>
-          <h6 class="card-subtitle mb-2 text-body-secondary"><span th:text="|${product.price}|"></span> <span th:utext="${'&#36;'+'&nbsp;'}"></span> <br> <span th:text="|${product.category}|"></span></h6>
+          <h6 class="card-subtitle mb-2 text-body-secondary"><span th:text="|${product.price}|"></span> <span th:utext="${'&#36;'+'&nbsp;'}"></span> <br> <span th:text="|${product.getCategoryString()}|"></span></h6>
```

Comment:
Avoid using \<br> like this, as it leads to potentially inconsistent formatting.
```suggestion
         <h6 class="card-subtitle mb-2 text-body-secondary"><span th:text="|${product.price}|"></span> <span th:utext="${'&#36;'+'&nbsp;'}"></span></h6>
          <h6 class="card-subtitle mb-2 text-body-secondary"><span th:text="|${product.getCategoryString()}|"></span></h6>
```